src/pages/App.jsx

```diff
@@ -8,7 +8,7 @@ import { Expenses } from './Expenses';
 import { Fallback } from './Fallback';
 import { Landing } from './Landing';
 import { Login } from './Login';
-
+import { WalletViewPage } from './WalletViewPage';
```

Comment:
Maybe 'WalletView' name for page will be more consistent in our current naming?
I can be wrong here.
@itsnotpolza @kalashnikau, your opinion?

And also add new line between import and export.
I know it's a little bit (um... too 😬) pedantic from my side, but nevertheless)

Reply:
Yep it will be more consistent with our current approach

Reply:
done, but I wanted the page and component to have  different names, so I changed name of the component too